dotCMS/src/main/java/com/dotcms/contenttype/model/type/VanityUrlContentType.java

```diff
@@ -0,0 +1,90 @@
+package com.dotcms.contenttype.model.type;
+
+import java.util.List;
+
+import com.dotmarketing.util.Config;
+import org.immutables.gson.Gson;
+import org.immutables.value.Value;
+
+import com.dotcms.contenttype.model.field.DataTypes;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.field.ImmutableCustomField;
+import com.dotcms.contenttype.model.field.ImmutableSelectField;
+import com.dotcms.contenttype.model.field.ImmutableTextField;
+import com.dotcms.repackage.com.google.common.collect.ImmutableList;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+/**
+ * Provides the basic outline of a Vanity URL Content Type in dotCMS. Vanity
+ * URLs are alternate reference paths to internal or external URL's. Vanity URLs
+ * are most commonly used to give visitors to the website a more user-friendly
+ * or memorable way of reaching an HTML page or File, that might actually live
+ * “buried” in a much deeper path.
+ * 
+ * @author Jose Castro
+ * @version 4.2.0
+ * @since May 25, 2017
+ *
+ */
+@JsonSerialize(as = ImmutableVanityUrlContentType.class)
+@JsonDeserialize(as = ImmutableVanityUrlContentType.class)
+@Gson.TypeAdapters
+@Value.Immutable
+public abstract class VanityUrlContentType extends ContentType implements Expireable, Multilinguable {
+
+	private static final long serialVersionUID = 1L;
+
+	private final String TITLE_FIELD_NAME = "Title";
+	public static final String TITLE_FIELD_VAR = "title";
+	private final String SITE_FIELD_NAME = "Site";
+	public static final String SITE_FIELD_VAR = "site";
+	private final String URI_FIELD_NAME = "URI";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)